[15.0][ADD] website_sale_stock_provisioning_time.

nikunjantala opened the pull request.

Commit:
[ADD] website_sale_category_empty: Create module (#342)

* [ADD] website_sale_category_empty: Create module
* Add module to hide categories that have no products associated with them

* [FIX] website_sale_hide_empty_category: PR Fix
* Add trailing comma to manifest
* Rename Module

eLBati committed:
porting website_sale_require_login to 12.0

remytms committed:
[MIG] website_sale_hide_empty_category: Migration to 11.0

SimoRubi committed:
[ADD] website_sale_order_type (#253)

sergio-teruel committed:
[11.0][ADD] website_sale_secondary_unit: New module

sergio-teruel committed:
[11.0][FIX] website_sale_secondary_unit: Crash if product_uom not come in values

yajo committed:
[FIX] website_sale_require_login: Fix public user test

This test was expecting sign up to be disabled because `auth_signup` is not in its module graph, but since that addon is autoinstallable, it could be installed without our knowledge.

Just make sure signup is disabled before running the public user test, to make it unitary.

Also, moved to at-install mode, since the post-install one adds no value.

sergio-teruel committed:
[11.0][FIX] website_sale_secondary_unit: Fix tour tests

sergio-teruel committed:
[11.0][IMP] website_sale_secondary_unit: Display product uom after price when product has secondary units defined

sergio-teruel committed:
[MIG] website_sale_secondary_unit: Migration to v12.0

wpichler committed:
[MIG] Migrated to Odoo 12

Tardo committed:
[IMP] website_sale_require_login: black, isort

Tardo committed:
[MIG] website_sale_require_login: Migration to 13.0

Tardo committed:
[IMP] website_sale_require_login: Readme

sergio-teruel committed:
[FIX] website_sale_secondary_unit: Fix quantity position in cart lines

CarlosRoca13 committed:
[IMP] : black, isort, prettier

CarlosRoca13 committed:
[MIG] website_sale_hide_empty_category: Migration to v13

Commit:
[ADD] website_sale_product_attachment: add attachments to products in website

This module lets you publish downloadable attachments in a product page.

This is useful if you want to publish firmwares, manuals, specs, warranties,
or whatever document related to the product.

@Tecnativa TT20984 TT23657

oca-travis committed:
[UPD] Update website_sale_product_attachment.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[FIX] website_sale_product_attachment: autogenerate index.html

OCA-git-bot committed:
website_sale_product_attachment 12.0.1.0.1

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_attachment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_attachment/

chienandalu committed:
[FIX] website_sale_product_attachment: role warning

OCA-git-bot committed:
website_sale_product_attachment 12.0.1.1.0

Commit:
Translated using Weblate (Spanish)

Currently translated at 100.0% (9 of 9 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_attachment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_attachment/es/

Commit:
[FIX] website_sale_product_attachment: sort attachments, support long names

Without this fix, the order in which attachments were displayed couldn't be predictable. Now, it's as expected by user: by name.

Also the thumbnails got stretched when the attachment had a long name.

@Tecnativa TT24437

OCA-git-bot committed:
website_sale_product_attachment 12.0.1.1.1

CarlosRoca13 committed:
[ADD] website_sale_wishlist_keep: New module that keep the items on wishlist when we add to cart

sergio-teruel committed:
[NEW] website_sale_tax_toggle: Display price with or without taxes

oca-travis committed:
[UPD] Update website_sale_tax_toggle.pot

sergio-teruel committed:
[NEW] website_sale_tax_toggle: Test more resilient to pricelists

sergio-teruel committed:
[NEW] website_sale_tax_toggle: Test more resilient to CI

sergio-teruel committed:
[IMP] website_sale_secondary_unit: Set default secondary uom for products when "Add to cart" view is enabled.

sergio-teruel committed:
[IMP] website_sale_tax_toggle: Fix tests

AndreVolksdorf committed:
Added translation using Weblate (German)

Commit:
[FIX] website_tax_toggle: make tour effective

This tour was not effective because:

- Steps did not have proper wait conditions.
- That made them run fast without actually testing the desired effect.
- That left many remaining requests that could made tests to fail randomly, as you can see in https://github.com/OCA/e-commerce/pull/420#issuecomment-672026443.

Now it's shorter but more effective:

- It skips going to `/shop` at the start because it starts already there.
- It moves tour code to demo data.
- It adds proper checks to each trigger, to make sure it doesn't run before checking the result is OK.

@Tecnativa TT24410

Commit:
[FIX] website_sale_tax_toggle: compatibilize tour with website_sale_b2x_alt_price

@Tecnativa TT24410

kh-badep committed:
Added translation using Weblate (French)

kh-badep committed:
Translated using Weblate (French)

Currently translated at 33.3% (3 of 9 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_attachment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_attachment/fr/

pedrobaeza committed:
[FIX] website_sale_*: Proper title: s/Wesbite/Website

dalonsod committed:
[ADD] website_sale_product_description

joao-p-marques committed:
[MIG] website_sale_order_type: Migration to 11.0

joao-p-marques committed:
[MIG] website_sale_order_type: Migration to 12.0

claudiagn committed:
Added translation using Weblate (Spanish)

Rad0van committed:
allow for recursive evaluation

Commit:
[FIX] website_sale_order_type: speed up tour

Without this patch, it's easy that the tour fails randomly due to a timeout when generating the sales order report. These assets can take a long time, increasing when more addons are installed.

In an integrated database, this patch makes the test go from this log:

    2020-11-06 10:34:53,941 34 INFO devel werkzeug: 127.0.0.1 - - [06/Nov/2020 10:34:53] "POST /payment/transfer/feedback HTTP/1.1" 302 - 730 4.368 11.800

To this one:

    2020-11-06 10:32:55,517 34 INFO devel werkzeug: 127.0.0.1 - - [06/Nov/2020 10:32:55] "POST /payment/transfer/feedback HTTP/1.1" 302 - 479 2.906 5.771

That's about 50% the time.

In the same patch, I remove the `self.env` patch that was only needed for Odoo v11 and lower.

chienandalu committed:
[ADD] website_sale_invoice_address: New module

ypapouin committed:
Added translation using Weblate (French)

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (2 of 2 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_tax_toggle
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_tax_toggle/fr/

ypapouin committed:
Translated using Weblate (French)

Currently translated at 100.0% (9 of 9 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_attachment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_attachment/fr/

ypapouin committed:
Added translation using Weblate (French)

bosd committed:
Added translation using Weblate (Dutch)

Currently translated at 100.0% (2 of 2 strings)

Translation: e-commerce-13.0/e-commerce-13.0-website_sale_hide_empty_category
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_hide_empty_category/nl/

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 33.3% (3 of 9 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_attachment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_attachment/nl/

joao-p-marques committed:
[FIX] website_sale_order_type: simplify tour

With the old tour, timeout problems started to occur, even with 7830084547e7df29c6dcd377dd4dc39a0319b594

This partially reverts that, and instead uses a simpler tour to test the creation of the SO.

TT26399

oca-travis committed:
[UPD] Update website_sale_product_description.pot

OCA-git-bot committed:
[ADD] icon.png

dalonsod committed:
Added translation using Weblate (Spanish)

dalonsod committed:
Added translation using Weblate (Spanish)

dalonsod committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (3 of 3 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_description
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_description/es/

claudiagn committed:
Added translation using Weblate (Spanish)

claudiagn committed:
Added translation using Weblate (Catalan)

claudiagn committed:
Added translation using Weblate (Catalan)

claudiagn committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (9 of 9 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_attachment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_attachment/ca/

claudiagn committed:
Added translation using Weblate (Catalan)

claudiagn committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (3 of 3 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_description
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_description/ca/

claudiagn committed:
Added translation using Weblate (Catalan)

claudiagn committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (2 of 2 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_tax_toggle
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_tax_toggle/ca/

claudiagn committed:
Added translation using Weblate (Catalan)

CarlosRoca13 committed:
[IMP] website_sale_wishlist_keep: black, isort, prettier

CarlosRoca13 committed:
[MIG] website_sale_wishlist_keep: Migration to v13.0

CarlosRoca13 committed:
[IMP] website_sale_secondary_unit: black, isort, prettier

sergio-teruel committed:
[IMP] website_sale_tax_toggle: black, isort

sergio-teruel committed:
[MIG] website_sale_tax_toggle: Migration to v13.0

CarlosRoca13 committed:
[MIG] website_sale_secondary_unit: Migration to v13.0

victoralmau committed:
[IMP] website_sale_product_attachment: black, isort, prettier

victoralmau committed:
[MIG] website_sale_product_attachment: Migration to 13.0

claudiagn committed:
Added translation using Weblate (Catalan)

Currently translated at 100.0% (2 of 2 strings)

Translation: e-commerce-13.0/e-commerce-13.0-website_sale_hide_empty_category
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_hide_empty_category/ca/

claudiagn committed:
Translated using Weblate (Spanish)

Currently translated at 100.0% (2 of 2 strings)

Translation: e-commerce-13.0/e-commerce-13.0-website_sale_hide_empty_category
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_hide_empty_category/es/

claudiagn committed:
Added translation using Weblate (Spanish)

claudiagn committed:
Added translation using Weblate (Catalan)

oca-travis committed:
[UPD] Update website_sale_product_attachment.pot

OCA-git-bot committed:
[UPD] README.rst

CarlosRoca13 committed:
[IMP] website_sale_tax_toggle: Making test more resilient

pedrobaeza committed:
[FIX] website_sale_secondary_unit: Adjust to upstream changes

As the product.secondary.unit view has been isolated, we have to
inherit from that one.

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (3 of 3 strings)

Translation: e-commerce-12.0/e-commerce-12.0-website_sale_product_description
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-12-0/e-commerce-12-0-website_sale_product_description/nl/

bosd committed:
Added translation using Weblate (Dutch)

bosd committed:
Translated using Weblate (Dutch)

Currently translated at 100.0% (2 of 2 strings)

Translation: e-commerce-13.0/e-commerce-13.0-website_sale_tax_toggle
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_tax_toggle/nl/

bosd committed:
[MIG] website_sale_product_description: Migration to 13.0

bosd committed:
[IMP] website_sale_product_description: black, isort, prettier

bosd committed:
[MIG] website_sale_product_description: Migration to 14.0

oca-travis committed:
[UPD] Update website_sale_product_description.pot

OCA-git-bot committed:
[UPD] README.rst

CarlosRoca13 committed:
[FIX] website_sale_secondary_unit: Fixed price computation

Before this changes, the change in the value of the secondary unit input field
didn't launch the execution of the onchanges of the real input field.

Adding this changes, we achieve to capture the event of the new input field
to launch the onchanges of the real input field.

cc @Tecnativa TT29708

chienandalu committed:
[FIX] webiste_sale_order_type: trigger onchanges

If we don't trigger the onchanges, we won't get some of the default
values that the order type has defined

TT30235

sergio-teruel committed:
[FIX] website_sale_secondary_unit: Remove sudo access to avoid to display unpublished secondary units.
TT31018

pedrocs-exo committed:
Added translation using Weblate (Portuguese)

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (6 of 6 strings)

Translation: e-commerce-14.0/e-commerce-14.0-website_sale_product_description
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-14-0/e-commerce-14-0-website_sale_product_description/pt/

celm1990 committed:
FIX: script for delete view before update
complementary to https://github.com/OCA/e-commerce/commit/3fa2331d53bda85c81cb6410b6ea0f35c21e6139

ibuioli committed:
Added translation using Weblate (Spanish (Argentina))

ibuioli committed:
Translated using Weblate (Spanish (Argentina))

Currently translated at 100.0% (6 of 6 strings)

Translation: e-commerce-14.0/e-commerce-14.0-website_sale_product_description
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-14-0/e-commerce-14-0-website_sale_product_description/es_AR/

oihane committed:
[IMP] website_sale_order_type: black, isort, prettier

oihane committed:
[MIG] website_sale_order_type: Migration to 13.0

pedrobaeza committed:
[FIX] website_sale_product_attachment: Read attachment metadata with sudo

In v13, there's no general read ACL for public files, so we need to get
filenames and file type using sudo. The download is performed normally
due to the public=True field in the attachments.

oihane committed:
[IMP] website_sale_order_type: black, isort, prettier

oihane committed:
[MIG] website_sale_order_type: Migration to 14.0

pedrobaeza committed:
[IMP] website_sale_product_attachment: Introduce website name

On previous version, attachments had 2 fields for adding both attachment
name and file name.

Now on v13, there's only one, that is fille with the file name. On
initial migration, it was considered that this field is enough, but
putting file names on the website product page can be ugly, limited and
confusing, so we are adding here a new field to store the name we want
to give it for the website e-commerce product page.

It also includes migration script for recovering the old information if
coming from v12.

oca-travis committed:
[UPD] Update website_sale_product_attachment.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-13.0/e-commerce-13.0-website_sale_product_attachment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_product_attachment/

chienandalu committed:
[IMP] website_sale_invoice_address: black, isort, prettier

chienandalu committed:
[MIG] website_sale_invoice_address: Migration to 13.0

TT28630

oca-travis committed:
[UPD] Update website_sale_invoice_address.pot

OCA-git-bot committed:
[UPD] README.rst

Commit:
[MIG] website_sale_require_login: Migration to 15.0

Noel000 committed:
Translated using Weblate (Catalan)

Currently translated at 100.0% (12 of 12 strings)

Translation: e-commerce-13.0/e-commerce-13.0-website_sale_product_attachment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-13-0/e-commerce-13-0-website_sale_product_attachment/ca/

CarlosRoca13 committed:
[IMP] website_sale_tax_toggle: black, isort, prettier

pedrobaeza committed:
[IMP] Update dotfiles + switch to GH actions

pedrobaeza committed:
[IMP] Update dotfiles + switch to GH actions

CarlosRoca13 committed:
[MIG] website_sale_tax_toggle: Migration to 14.0

CarlosRoca13 committed:
[IMP] website_sale_tax_toggle: black, isort, prettier

CarlosRoca13 committed:
[MIG] website_sale_tax_toggle: Migration to 15.0

CarlosRoca13 committed:
[IMP] website_sale_hide_empty_category: black, isort, prettier

CarlosRoca13 committed:
[MIG] website_sale_hide_empty_category: Migration to 15.0

CarlosRoca13 committed:
[FIX] website_sale_require_login: Fix tests

CarlosRoca13 committed:
[IMP] website_sale_secondary_unit: black, isort, prettier

CarlosRoca13 committed:
[MIG] website_sale_secondary_unit: Migration to 15.0

github-user-name committed:
[15.0][ADD] website_sale_stock_provisioning_time.

pedrobaeza commented:
There's already https://github.com/OCA/e-commerce/tree/13.0/website_sale_stock_provisioning_date. Migrate it instead.

jans23 commented:
Both modules work a bit different. With module website_sale_stock_provisioning_date we couldn't display the expected provisioning time (in work days) on a particular product and it requires the expected stock move to be planned in Odoo. In our workflow we don't have such expected stock move but for some products we require several days to prepare an individual order before it can be shipped. We didn't find website_sale_stock_provisioning_date suitable to reflect such.

github-user-name committed:
Fix precommit code.

github-user-name committed:
[Add] README.rst in website_sale_stock_provisioning_time.

pedrobaeza commented:
OK, please reflect such differences in the README.

github-user-name committed:
[Fix] readme DESCRIPTION.rst.

manengel commented on website_sale_stock_provisioning_time/models/product_template.py:
```diff
@@ -0,0 +1,9 @@
+    _inherit = "product.template"
+
+    provisioning_time_in_workdays = fields.Integer(
+        string="Provisioning time in workdays"
```
workdays --> work days I think.

jans23 replied:
It seems both to be ok:

* https://dict.leo.org/englisch-deutsch/arbeitstage
* https://www.dict.cc/?s=arbeitstage
* https://www.deepl.com/translator#de/en/arbeitstage

manengel replied:
O.K. then the module does look good to me.

OCA-git-bot committed:
Merge PR #647 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_require_login.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

Commit:
[IMP] website_sale_product_description: black, isort, prettier

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_require_login
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_require_login/

Commit:
[15.0][MIG] website_sale_product_description: migration to 15.0

OCA-git-bot committed:
Merge PR #666 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_comparison_hide_price.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
Merge PR #650 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_checkout_country_vat.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
Merge PR #654 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_checkout_skip_payment.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
Merge PR #645 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_hide_empty_category.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #671 into 15.0

Signed-off-by dreispt

Commit:
[UPD] Update website_sale_product_description.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #652 into 15.0

Signed-off-by dreispt

Commit:
[UPD] Update website_sale_secondary_unit.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
Merge PR #656 into 15.0

Signed-off-by ivantodorovich

Commit:
[UPD] Update website_sale_cart_expire.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

CarlosRoca13 committed:
[IMP] website_sale_wishlist_keep: black, isort, prettier

CarlosRoca13 committed:
[MIG] website_sale_wishlist_keep: Migration to 15.0

OCA-git-bot committed:
Merge PR #691 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_wishlist_keep.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_wishlist_keep
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_wishlist_keep/

ajaniszewska-dev committed:
Sync repo template

simahawk committed:
Merge pull request #692 from ajaniszewska-dev/sync_template

Sync repo template

OCA-git-bot committed:
[UPD] addons table in README.md

victoralmau committed:
[IMP] website_sale_suggest_create_account: Avoid deprecation warning with `Environment.manage()`

victoralmau committed:
[IMP] website_sale_require_login: Avoid deprecation warning with `Environment.manage()`

pedrobaeza committed:
[FIX] Update copier template

To avoid the Flake8 Gitlab problem.

pedrobaeza committed:
Merge pull request #698 from Tecnativa/15.0-copier

[15.0][FIX] Update copier template

OCA-git-bot committed:
Merge PR #697 into 15.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
website_sale_require_login 15.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
Merge PR #696 into 15.0

Signed-off-by pedrobaeza

OCA-git-bot committed:
website_sale_suggest_create_account 15.0.1.0.1

OCA-git-bot committed:
[UPD] addons table in README.md

Commit:
[IMP] website_sale_product_attachment: black, isort, prettier

Commit:
[MIG] website_sale_product_attachment: Migration to 15.0

OCA-git-bot committed:
[IMP] update dotfiles [ci skip]

stefan-tecnativa committed:
[IMP] website_sale_order_type: black, isort, prettier

pedrocs-exo committed:
Translated using Weblate (Portuguese)

Currently translated at 94.1% (16 of 17 strings)

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_checkout_skip_payment
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_checkout_skip_payment/pt/

stefan-tecnativa committed:
[15.0][MIG] website_sale_order_type: Migration to version 15.0

TT37013

OCA-git-bot committed:
Merge PR #706 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_cart_expire.pot

Commit:
[UPD] Update website_sale_order_type.pot

Commit:
[UPD] Update website_sale_secondary_unit.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_secondary_unit
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_secondary_unit/

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_order_type
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_order_type/

OCA-git-bot committed:
Merge PR #643 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_product_detail_attribute_image.pot

Commit:
[UPD] Update website_sale_product_detail_attribute_value_image.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_product_detail_attribute_value_image
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_product_detail_attribute_value_image/

OCA-git-bot committed:
Merge PR #642 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_tax_toggle.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_tax_toggle
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_tax_toggle/

pilarvargas-tecnativa committed:
[MIG] website_sale_invoice_address: Migration to version 15.0

TT37012

OCA-git-bot committed:
Merge PR #711 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_invoice_address.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #699 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_product_attachment.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
[UPD] README.rst

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_invoice_address
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_invoice_address/

OCA-git-bot committed:
Merge PR #644 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_product_sort.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

OCA-git-bot committed:
Merge PR #649 into 15.0

Signed-off-by pedrobaeza

Commit:
[UPD] Update website_sale_b2x_alt_price.pot

OCA-git-bot committed:
[UPD] addons table in README.md

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

weblate committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: e-commerce-15.0/e-commerce-15.0-website_sale_product_sort
Translate-URL: https://translation.odoo-community.org/projects/e-commerce-15-0/e-commerce-15-0-website_sale_product_sort/

dsolanki-initos committed:
[CHANGE] update pre-commit.yaml

dsolanki-initos committed:
Merge branch '15.0-add-website-sale-stock-provisioning-time' of https://github.com/Nitrokey/odoo-e-commerce into 15.0-add-website-sale-stock-provisioning-time

pedrobaeza commented:
This pull request is incorrect containing a lot of undesired changes. Closing, and if re-submitted, it should be with another name that don't collapse with the existing one.
